R/describe_missing.R

```diff
@@ -0,0 +1,115 @@
+#' @title Describe Missing Values in Data According to Guidelines
+#'
+#' @description Provides a detailed description of missing values in a data frame.
+#' This function reports both absolute and percentage missing values of specified
+#' column lists or scales, following recommended guidelines. Some authors recommend
+#' reporting item-level missingness per scale, as well as a participant's maximum
+#' number of missing items by scale. For example, Parent (2013) writes:
+#'
+#' *I recommend that authors (a) state their tolerance level for missing data by scale
+#' or subscale (e.g., "We calculated means for all subscales on which participants gave
+#' at least 75% complete data") and then (b) report the individual missingness rates
+#' by scale per data point (i.e., the number of missing values out of all data points
+#' on that scale for all participants) and the maximum by participant (e.g., "For Attachment
+#' Anxiety, a total of 4 missing data points out of 100 were observed, with no participant
+#' missing more than a single data point").*
+#'
+#' @param data The data frame to be analyzed.
+#' @param vars Variable (or lists of variables) to check for missing values (NAs).
```

Comment:
We use `select`, `exclude`, etc. in all other dataframe functions, I think we should here as well.

Reply:
Here it works a little bit differently than `select` elsewhere. `vars` takes a list of list of strings (such as `list(c("Ozone", "Solar.R", "Wind"), c("Temp", "Month", "Day"))`) to take into account the nested structure of the items / columns. I can rename it to `select`, but do you think it will create confusion or expectations that it should rely on and work with `.select_nse`? Or should we include `select` and `exclude` in addition to `vars`? I'm not sure how `.select_nse` could accommodate the nested structure like I'm doing right now 🤔